translations/en_GB/atrium-translations-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionOptionalAssertion.kt

```diff
@@ -0,0 +1,8 @@
+package ch.tutteli.atrium.translations
+
+import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
+
+enum class DescriptionOptionalAssertion(override val value: String) : StringBasedTranslatable {
+    EMPTY("empty"),
+    NOT_EMPTY("not empty"),
```

Comment:
please remove NOT_EMPTY will not be used

Reply:
Removed